netket/graph/common_lattices.py

```diff
@@ -162,6 +164,8 @@ def Cube(length: int, *, pbc: bool = True) -> Lattice:
         >>> print(g.n_nodes)
         1000
     """
+    if not isinstance(length, int) or length <= 0:
+        raise TypeError("Argument `length` must be a positive integer")
```

Comment:
I think we do not need this because it calls hypercube?

Reply:
Good point.